01-setup-ci/readme.md

```diff
@@ -83,7 +83,8 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-+     - uses: actions/checkout@v3
```

Comment:
Remove `+   uses: actions/checkout@v3` bellow includes version 4

Reply:
It is removed :) But it is hard to check differences on a diff statement. I attach a screenshot to see it more clear.

![image](https://github.com/Lemoncode/github-actions-playground/assets/33926302/39e0ea3b-6e74-4fd9-b498-bf669cc11d54)